includes/ApplHandlingByStr.php

```diff
@@ -33,6 +34,46 @@ function showStatus( $stat_str )
 		{
 			echo "<td><b>REJECTED</b> by ".$str[0] ."</td>";
 		}
+		else if( $str[1] == 'F' )
+		{
+			echo "<td><b>FORWARDED</b> from ".$str[0]."</td>";
+		}
+		else
+		{
+			echo "<td><b>Unknown Status</b></td>";
+		}
+	}
+}
+
+
+function showFullStatus( $stat_str )
+{
+	$STR = explode ( ';', $stat_str );
+	for ( $i =  1;  $i < count ( $STR ) ; $i++ )
+	{
+		$str = explode ( ',', $STR[$i] );
+		$usr = $str[0];
+		
+		if( $str[1] == 'P' )
+		{
+			echo "<td><b>PENDING</b> by $usr</td>";
+		}
+		else if( $str[1] == 'A' )
+		{
+			echo "<td><b>APPROVED</b> by ".$str[0] ."</td>";
+		}
+		else if( $str[1] == 'Y' )
+		{
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->